ios/TabViewProvider.swift

```diff
@@ -154,12 +175,23 @@ struct TabData: Codable {
   }
 }
 
-extension UIImage {
-  func resizeImageTo(size: CGSize) -> UIImage? {
+extension PlatformImage {
+  func resizeImageTo(size: CGSize) -> PlatformImage? {
+#if os(iOS)
     UIGraphicsBeginImageContextWithOptions(size, false, 0.0)
-    self.draw(in: CGRect(origin: CGPoint.zero, size: size))
-    let resizedImage = UIGraphicsGetImageFromCurrentImageContext()!
+    self.draw(in: CGRect(origin: .zero, size: size))
+    let resizedImage = UIGraphicsGetImageFromCurrentImageContext()
     UIGraphicsEndImageContext()
     return resizedImage
+#elseif os(macOS)
+    let newImage = NSImage(size: size)
+    newImage.lockFocus()
+    self.draw(in: CGRect(origin: .zero, size: size),
+              from: CGRect(origin: .zero, size: self.size),
+              operation: .copy,
+              fraction: 1.0)
+    newImage.unlockFocus()
+    return newImage
+#endif
```

Comment:
See a similar change here: https://github.com/microsoft/react-native-macos/pull/2209
`lockFocus` and that pattern is deprecated in favor of [imageWithSize:flipped:drawingHandler:](https://developer.apple.com/documentation/appkit/nsimage/1519860-imagewithsize). Similarly, I think `UIGraphicsBeginImageContextWithOptions` is mostly replaced with `UIGraphicsImageRenderer`

Reply:
Thanks! I will fix it